.pre-commit-config.yaml

```diff
@@ -30,3 +26,10 @@ repos:
         files: 'Explorer/.*\.(js|jsx|ts|tsx)$'
         entry: bash -c 'cd Explorer && npm run precommit-lint "$@"'
         types: [file]
+      - id: golangci-lint
+        name: golangci-lint
+        language: system
+        types: [file, go]
+        # This checks all files instead of only the ones that were changed.
```

Comment:
This should mean the output of the hook should not differ between `pre-commit run golangci-lint` and `pre-commit run golangci-lint -a` should be the same, which is not the case  for me.